src/components/content/heroes/aurora/weather-easter-egg/precipitation-effect.tsx

```diff
@@ -13,12 +13,12 @@ interface RaindropProps {
 interface SnowflakeProps {
   fallDuration: number;
   size: number;
-  animation: any;
   blur: number;
   transparency: number;
   left: number;
   top: number;
   animation_delay: number;
+  snowflake?: PrecipitationData;
```

Comment:
What is the reason for adding the snowflake here? If I understand this correctly, `snowflake` is only used in `swayAnimation`. `swayAnimation` only needs `left` & `fallDuration`, which are both already in the props.